_resources/port1.0/group/mpi-1.0.tcl

```diff
@@ -309,11 +309,14 @@ proc mpi.setup {args} {
         if {${os.major} <= 12 && $is_mpich} {
             lappend disabled -clang60 -clang70 -clang80 -clang90
         }
+        if {!$is_mpich} {
```

Comment:
Should add `|| ${os.major} < 10` to the condition, to match the mpich portfile.

Reply:
$ ssh-keygen -t ed25519 -C "your_email@example.com"